### PR TITLE
Require family name to be set into order to set standalone course

### PR DESCRIPTION
**Pull request by dmcavoy**
Requires that you select a family name before you can check that a unit is a standalone course. Added help text to the existing help tip.

Added a test to check that you can't set the standalone course until you set family name.

Completes [PLAT-450](https://codedotorg.atlassian.net/browse/PLAT-450)

stand-alone-course

### CODE REVIEW

File: apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx [hunk 1]
```diff
@@ -114,9 +114,8 @@ export default class ScriptEditor extends React.Component {
       .removeAttr('selected', true);
   };
 
-  handleUmbrellaSelectChange = event => {
```
Comment (dmcavoy) [inline apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx:117]: I noticed that the state for curriculumUmbrella was not being used so I removed it.

File: apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx [hunk 2]
```diff
@@ -413,6 +412,7 @@ export default class ScriptEditor extends React.Component {
                   name="is_course"
                   type="checkbox"
                   defaultChecked={this.props.isCourse}
+                  disabled={!this.state.familyName}
```
Comment (bethanyaconnor) [inline apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx]: What happens if you pick a `familyName`, select this checkbox, then set `familyName` to `(None)`? This checkbox would be disabled and couldn't be unchecked right?

Comment (dmcavoy) [reply apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx]: Oh good call. Will need to look into that.

**Review by davidsbailey — APPROVED** [review]
LGTM after addressing Bethany's feedback.

**Pull request merged**